torch/csrc/jit/operator_upgraders/version_map.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+#include <string>
+#include <unordered_map>
+#include <vector>
+
```

Comment:
same as upgraders.h

Reply:
As per discussion, this file will be static and will be merged with the other stack of diffs I have